documentation/docs/14-configuration.md

```diff
@@ -138,6 +139,12 @@ When pages are prerendered, the CSP header is added via a `<meta http-equiv>` ta
 
 > When `mode` is `'auto'`, SvelteKit will use nonces for dynamically rendered pages and hashes for prerendered pages. Using nonces with prerendered pages is insecure and therefore forbidden.
 
+# link_header
+
+When `true` the server will inject a `Link: </my-files.js>; rel=preload` header defined by [MDN Link](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Link). This can be used to let the browser know about modulepreloads before the html is parsed.
+
+> Cloudflare with [Early Hits](https://developers.cloudflare.com/cache/about/early-hints) enabled can then further optimize this by fetching these assets as it flies through the edge.
```

Comment:
I don't think it's worth mentioning early hints here — since we return the response body along with the headers, I'd be very surprised if early hints offers anything over and above a `Link` header with an ordinary response

```suggestion
When `true` the server will inject a [`Link` header](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Link). This can be used to let the browser eagerly fetch essential resources before the HTML is parsed.
```